lms/djangoapps/courseware/tests/test_video_handlers.py

```diff
@@ -1289,7 +1292,7 @@ def test_en_with_empty_sub(self):
             """)
 
         self.assertEqual(text, expected_text)
-        self.assertEqual(filename, self.item.youtube_id_1_0 + '.srt')
+        self.assertEqual(filename, 'en_' + self.item.youtube_id_1_0 + '.srt')
```

Comment:
This kind of change seems harmless, but unknown what other functionality depends on the transcript name. Things were previously inconsistent anyway with the two different functions returning different naming formats - either way we should consolidate to either `rawid` or `en_rawid.srt` style.

Reply:
Agreed.  And it should be the more explicit style that includes the language, even if it's the default one.

Reply:
Can this be modified so that it does not break backwards compatibility? People do run scripts, and anything that alters the serialization is something that needs to be messaged much more broadly. If this is going to break compatibility by altering the output file names, what things will it break?

Reply:
@ormsbee, the trouble is that some transcript functions return the language code prefix in the filename, while others didn't. There isn't a way to properly consolidate this without breaking backwards compatibility with at least one of the use cases. I can update it to not return the language code, if you confirm that is the desired behaviour.

Reply:
Actually, does this affect the export or only the file download in the view? If it's the view handler, I don't care.

Reply:
@ormsbee 

> does this affect the export or only the file download in the view?

I dug a little deeper - the only transcript method which returned without the language code prefix in the filename, was the removed `VideoTranscriptsMixin.get_transcript` method. The only place in the codebase which used this method was `VideoBlock.index_dictionary`, and that ignored the output filename. I believe we're pretty safe to change this.

I guess it's always possible that someone was using `VideoBlock.get_transcript` in a script, but this is unlikely, and they should switch to the newer `transcripts_utils.gen_transcript` function anyway.

CC @arbrandes

Reply:
Sounds good then. Please squash the commits and add the context from the PR message to the commit message, and I'll merge.

Reply:
Thanks @ormsbee, squashed and commit message updated.